thermo/data/co2-thermo.yaml

```diff
@@ -0,0 +1,62 @@
+description: |-
+  CO2 phases based on the equation of state 
+
+cantera-version: 2.5.0a3
+date: Wed, 10 Jul 2020 11:38:00 +0000
+
+units: {length: cm, time: s, quantity: mol, activation-energy: cal/mol}
+
+phases:
+- name: CO2-Ideal
+  thermo: ideal-gas
+  elements: [O, C]
+  species: [CO2]
+  kinetics: gas
+  transport: mixture-averaged
+  state: {T: 300.0, P: 1 atm}
+
+- name: CO2-RK
+  thermo: Redlich-Kwong
+  elements: [O, C]
+  species: [{rk-species: [CO2]}]
+  kinetics: gas
+  transport: mixture-averaged
+  state: {T: 300.0, P: 1 atm}
+
+species:
+- name: CO2
+  composition: {C: 1, O: 2}
+  thermo:
+    model: NASA7
+    temperature-ranges: [200.0, 1000.0, 3500.0]
+    data:
+    - [2.35677352, 0.00898459677, -7.12356269e-06, 2.45919022e-09, -1.43699548e-13,
+      -48371.9697, 9.90105222]
+    - [3.85746029, 0.00441437026, -2.21481404e-06, 5.23490188e-10, -4.72084164e-14,
+      -48759.166, 2.27163806]
+    note: L7/88
+  transport:
+    model: gas
+    geometry: linear
+    well-depth: 244.0
+    diameter: 3.763
+    polarizability: 2.65
+    rotational-relaxation: 2.1
+  
+rk-species:
+- name: CO2
+  composition: {C: 1, O: 2}
+  thermo:
+    model: NASA7
+    temperature-ranges: [200.0, 1000.0, 3500.0]
+    data:
+    - [2.35677352, 0.00898459677, -7.12356269e-06, 2.45919022e-09, -1.43699548e-13,
+      -48371.9697, 9.90105222]
+    - [3.85746029, 0.00441437026, -2.21481404e-06, 5.23490188e-10, -4.72084164e-14,
+      -48759.166, 2.27163806]
+  equation-of-state:
+    model: Redlich-Kwong
+    units: {length: cm, pressure: bar, quantity: mol}
+    a: [7.54e7, -4.13e4]
+    b: 27.80
+    
```

Comment:
Make sure there are no extra spaces on this line.

Reply:
Cleaned up.